### PR TITLE
feat: organization page made species plants in list

### PR DESCRIPTION
**Pull request by Harshalyadav**
# Description

Species plants are shown in the list

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/pages/organizations/[organizationid].js [hunk 1]
```diff
@@ -357,17 +357,20 @@ export default function Organization(props) {
...
               }}
             >
               {organization?.species?.species?.map((s) => (
                 <React.Fragment key={s.name}>
-                  <TreeSpeciesCard
-                    name={s.name}
-                    subTitle={s.desc || '---'}
-                    count={s.total}
-                  />
+                  <li key={s.id}>
```
Comment (RubenSmn) [inline src/pages/organizations/[organizationid].js:367]: No need for a key here since this is not the _top level_ component in the loop.

**Review by RubenSmn — CHANGES_REQUESTED** [review]
We should also convert the [_hired planters_](https://github.com/Greenstand/treetracker-web-map-client/pull/1541/files#diff-1160c2738fc888e43168fd35336e393916c22bb258e8c174a4bfeee63d0fd7c3R385-R431) section to a list. We can easily use the `component` prop for this.

Also we might need to fix the default styling from the lists. Take a look at this [PR](https://github.com/Greenstand/treetracker-web-map-client/pull/1540)

File: src/pages/organizations/[organizationid].js (hunk 1 above)
Comment (Harshalyadav) [reply src/pages/organizations/[organizationid].js:367]: Ok, I see

**Comment by RubenSmn** [comment]
Resolved by #1559

**Pull request closed without merging**